tests/test_ilastik/widgets/test_featureTableWidget.py

```diff
@@ -0,0 +1,43 @@
+from unittest import mock
+
+import pytest
+from PyQt5.QtCore import Qt
+
+from ilastik.widgets.featureTableWidget import FeatureTableWidget, FeatureEntry
+
+
+@pytest.fixture
+def widget(qtbot):
+    w = FeatureTableWidget(
+        None,
+        (
+            ("Color", [FeatureEntry("Banana", minimum_scale=0.3)]),
+            ("Edge", [FeatureEntry("Mango"), FeatureEntry("Cherry")]),
+        ),
+        [0.3, 0.7, 1, 1.6, 3.5, 5.0, 10.0],
+        computeIn2d=[True, True, True, True, True, True, True],
```

Comment:
For the `computeIn2d` it would be interesting to have at least _some_ `False` values in place. Previous incarnation of "Select all" would set them all to `True`

Reply:
Updated to use a mix of the feature dimensionality.